@xen-orchestra/web/src/components/host/system/HostSystemGeneralInformation.vue

```diff
@@ -3,51 +3,53 @@
     <UiTitle>
       {{ t('general-information') }}
     </UiTitle>
-    <VtsQuickInfoRow :label="t('name')" :value="host.name_label" />
-    <VtsQuickInfoRow :label="t('uuid')" :value="host.id" />
-    <VtsQuickInfoRow :label="t('description')" :value="host.name_description" />
-    <VtsQuickInfoRow :label="t('tags')">
-      <template v-if="host.tags.length > 0" #value>
-        <UiTagsList>
-          <UiTag v-for="tag in host.tags" :key="tag" accent="info" variant="secondary">
-            {{ tag }}
-          </UiTag>
-        </UiTagsList>
-      </template>
-    </VtsQuickInfoRow>
-    <VtsQuickInfoRow :label="t('status')">
+    <UiLabelValue :label="t('name')" :value="host.name_label" />
+    <UiLabelValue :label="t('uuid')" :value="host.id" />
+    <UiLabelValue :label="t('description')" :value="host.name_description" />
+    <UiLabelValue :label="t('tags')" :value="host.tags" />
+    <UiLabelValue :label="t('status')">
       <template #value>
         <VtsStatus :status="host.enabled" />
       </template>
-    </VtsQuickInfoRow>
-    <VtsQuickInfoRow :label="t('pool')">
+    </UiLabelValue>
+    <UiLabelValue :label="t('pool')">
       <template v-if="pool !== undefined" #value>
-        <UiLink size="medium" :to="`/pool/${pool.id}/`" icon="fa:city">
-          {{ pool.name_label }}
+        <UiLink size="medium" :to="`/pool/${pool.id}/`" icon="fa:city" class="link">
```

Comment:
```suggestion
        <UiLink size="medium" :to="`/pool/${pool.id}/`" icon="fa:city">
```

I think you can remove this class